packages/kbn-shared-ux-components/src/empty_state/empty_state_page.mdx

```diff
@@ -0,0 +1,23 @@
+---
+id: sharedUX/Components/EmptyStatePage
+slug: /shared-ux-components/empty_state_page
+title: Empty State Page
+summary: A page to be displayed when there is no data in Elasticsearch, or no data views
+tags: ['shared-ux', 'component']
+date: 2022-04-20
+---
+
+## Description
+
+Many plugins display "no data" page, either when there is no data in Elasticsearch, or there haven't been any data views created yet. This component is meant
+to be used in those scenarios. It displays an appropriate message to the user and facilitate addition of integrations and creation of data views. 
+
+## Component: `EmptyStatePage`
```

Comment:
Re: **Naming**

Is this component meant to be used directly by plugins? If so, we definitely should prefix it with `Kibana` as to signify that that is a platform-wide component.

As for `EmptyState`, I agree that it's confusing as to what "Empty" specifically means in this context. So the question will be whether this is will also end up handling specifically just no-saved-objects state as well? Like an empty listing page. If so, then I think `Empty` should be ok in this context. Otherwise, you might consider `KibanaNoDataPage` as it reads to me as a parent to `NoDataConfigPage` and `NoDataViewPage` which are just specifics of `NoData`.

Reply:
Yes, this component can be used directly by plugins, it's not a part of any other component. I would call it `KibanaNoDataPage` then.